Dockerfile

```diff
@@ -1,7 +1,7 @@
 # This is a Dockerfile definition for Experimental Docker builds.
 # DockerHub: https://hub.docker.com/r/jenkins/jenkins-experimental/
 # If you are looking for official images, see https://github.com/jenkinsci/docker
-FROM maven:3.5.4-jdk-8 as builder
+FROM openjdk:8-jdk as builder
 
 COPY .mvn/ /jenkins/src/.mvn/
```

Comment:
```suggestion
COPY .mvn/ /jenkins/src/.mvn/
COPY mvnw /jenkins/src/mvnw
```

Reply:
tbh I don't think this Dockerfile is used and should prob just be removed, no one is maintaining it.